backend/test_auth.py

```
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
```

Ruff (0.14.7)

Shebang is present but file is not executable

(EXE001)